praw/models/reddit/live.py

```diff
@@ -554,6 +554,11 @@ class LiveUpdate(FullnameMixin, RedditBase):
 
     STR_FIELD = "id"
 
+    @property
```

Comment:
Can this simply be an attribute when it's not dynamic?

Reply:
It’s meant to override the property in the base class (`FullnameMixin`). It’s a property of course because others that implement it are dynamic.

If this were made a class attribute instead, would that not make it inconsistent with the interface `FullnameMixin` provides? If it were made a class attribute instead then it would be assignable. That’s the only difference it would make.

I originally had them all as a class attributes until @jarhill0 asked me to change it. It does make sense for them to be class attributes now though since it there’s no longer ever a need to configure them. That should be done only when we’re prepared to document that change though.

Reply:
Coveralls is complaining on the coverage of the `kind` property in the base class so I’m just going to make it an attribute there and here.